include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -173,11 +173,15 @@ namespace aspect
                                      const std::vector<double> &volume_fractions,
                                      const bool plastic_yielding,
                                      const MaterialModel::MaterialModelInputs<dim> &in,
-                                     MaterialModel::MaterialModelOutputs<dim> &out) const;
+                                     MaterialModel::MaterialModelOutputs<dim> &out,
+                                     const std::vector<double> &phase_function_values = std::vector<double>(),
+                                     const std::vector<unsigned int> &n_phases_per_composition =
+                                       std::vector<unsigned int>()) const;
+
 
           /**
-           * Reference viscosity used by material models using this rheology.
-           */
+          * Reference viscosity used by material models using this rheology.
```

Comment:
Please undo the removal of the space before the asterix. It should look like this:
```
/**
 *
 */
```